msal/build.gradle

```diff
@@ -184,8 +184,6 @@ task sourcesJar(type: Jar) {
 String commonVersion = project.hasProperty("distCommonVersion") ? project.distCommonVersion : "0.0.+"
 
 dependencies {
-    testImplementation project(path: ':common4j')
```

Comment:
why was there a testImplementation project(path: ':common4j') and testImplementation(testFixtures(project(":common4j"))) before? Now I see we have removed this.

Reply:
We don't need testImplementation project(path: ':common4j'), as we are only using testFixtures from common4j.